### PR TITLE
[BLD-1214] Add cosmic support

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: percona-packaging/scripts/psmdb_builder.sh [hunk 1]
```diff
@@ -239,15 +239,19 @@ install_gcc_54_deb(){
 set_compiler(){
     if [ x"${DEBIAN}" = xjessie -o x"${DEBIAN}" = xwheezy -o x"${DEBIAN}" = xtrusty -o x"${DEBIAN}" = xxenial ]; then
         export CC=/usr/local/gcc-5.4.0/bin/gcc-5.4
-	    export CXX=/usr/local/gcc-5.4.0/bin/g++-5.4
+            export CXX=/usr/local/gcc-5.4.0/bin/g++-5.4
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: indentation?

File: percona-packaging/scripts/psmdb_builder.sh [hunk 2]
```diff
@@ -239,15 +239,19 @@ install_gcc_54_deb(){
...
     fi
     if [ x"${DEBIAN}" = xstretch ]; then
         export CC=/usr/bin/gcc-6
-	    export CXX=/usr/bin/g++-6
+            export CXX=/usr/bin/g++-6
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: same

File: percona-packaging/scripts/psmdb_builder.sh (hunk 1 above)
Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: fixed

File: percona-packaging/scripts/psmdb_builder.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: fixed

**Pull request merged**